hub/powertoys/text-extractor.md

```diff
@@ -31,3 +36,65 @@ From the Settings menu, the following options can be configured:
 | Setting | Description |
 | :--- | :--- |
 | Activation shortcut | The customizable keyboard command to turn on or off this module. |
+
+## Supported languages
+
+Text Extractor can only recognize languages that have the OCR language pack installed.
+
+The list can be obtained via PowerShell by running the following commands:
+```powershell
+PS C:\Windows\system32> [Windows.Media.Ocr.OcrEngine, Windows.Foundation, ContentType = WindowsRuntime]
+PS C:\Windows\system32> [Windows.Media.Ocr.OcrEngine]::AvailableRecognizerLanguages
+```
+### How to query for OCR language packs
+
+The next command returns the list (PowerShell run as Administrator):
+```powershell
+PS C:\Windows\system32> Get-WindowsCapability -Online | Where-Object { $_.Name -Like 'Language.OCR*' }
+```
+
+An example output:
+```console
+....
+Name  : Language.OCR~~~el-GR~0.0.1.0
+State : NotPresent
+
+Name  : Language.OCR~~~en-GB~0.0.1.0
+State : NotPresent
+
+Name  : Language.OCR~~~en-US~0.0.1.0
+State : Installed
+
+Name  : Language.OCR~~~es-ES~0.0.1.0
+State : NotPresent
+
+Name  : Language.OCR~~~es-MX~0.0.1.0
+State : NotPresent
+....
+```
+
+If a language is not available in the output, then it's not supported by OCR.
+
+### How to install an OCR language pack
+
+The following commands install the OCR pack for "en-US":
+```powershell
+PS C:\Windows\system32> $Capability = Get-WindowsCapability -Online | Where-Object { $_.Name -Like 'Language.OCR*en-US*' }
```

Comment:
The c: stuff needs to be removed. My suggestion had this removal included.

Reply:
Sorry, missed that part. Removed the PS C:\Windows\system32> prefix as well.